app/models/importers/organisation.rb

```diff
@@ -13,8 +13,14 @@ def run
     loop do
       result = search_content_items_for_organisation
       result.each do |content_item_attributes|
-        attributes = content_item_attributes.slice('content_id')
-        organisation.content_items << ContentItem.new(attributes)
+        content_id = content_item_attributes['content_id']
+        link = content_item_attributes['link']
+
+        if content_id
```

Comment:
Lines 20 and 21 feels to be doing too much. It is not very easy to read

I would go with something similar to:

```ruby
        if content_id.present?
          link = content_item_attributes['link']
          attributes = content_item_attributes.slice('content_id')
          attributes.merge!(last_updated_at: find_public_updated_at(link))

          organisation.content_items << ContentItem.new(attributes)
        else
          Logger.warn('There is not content_id for ', slug)
        end
```

Although, in incoming commits we will extract a lot of logic from this class